cores/esp8266/Updater.cpp

```diff
@@ -360,18 +360,27 @@ size_t UpdaterClass::writeStream(Stream &data) {
         return 0;
     }
 
+    pinMode(2, OUTPUT);
```

Comment:
LED_BUILTIN is preferred here.

Also pin state should be backuped in case OTA fails and sketch should go on,
or,
The led pin (and also it's lit state which can be 0 for ON) could be passed through arguments for users who are using the led pin for something else like a relay.